packages/salesforcedx-utils-vscode/src/index.ts

```diff
@@ -1,9 +1,29 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
 export {
-  SFDX_PROJECT_FILE,
-  ENV_SFDX_DEFAULTUSERNAME,
-  ENV_SFDX_INSTANCE_URL,
-  SFDX_CONFIG_ISV_DEBUGGER_SID,
-  SFDX_CONFIG_ISV_DEBUGGER_URL,
-  DEFAULT_CONNECTION_TIMEOUT_MS,
-  CLIENT_ID
-} from './constants';
+  CompositeParametersGatherer,
+  EmptyParametersGatherer
+} from './commands/parameterGatherers';
+export { EmptyPostChecker } from './commands/postconditionCheckers';
+export {
+  EmptyPreChecker,
+  SfdxWorkspaceChecker
+} from './commands/preconditionCheckers';
+export {
+  CommandletExecutor,
+  SfdxCommandlet,
+  SfdxCommandletExecutor
+} from './commands/sfdxCommandlet';
```

Comment:
Export commandlet and telemetryService from index.ts since commandlet depends on telemetryService.

Reply:
Does this mean we should remove `index.ts` from the telemetry folder and probably have webpack not process it directly ?

Reply:
I've removed the contents of `telemetry/index.ts`: https://github.com/forcedotcom/salesforcedx-vscode/blob/8605f1f540c4a6bb24dbdb4a96e4a24a81159c04/packages/salesforcedx-utils-vscode/src/telemetry/index.ts 

Unit tests would need some adjustments too.